tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -54,6 +57,13 @@ def add_names_to_tensors(model: InputModel, places: List[Place]):
                 log.warn('Could not add an additional name to a tensor pointed to by \'{}\'. Details: {}'.format(
                     new_input['input_name'], str(e)))
 
+    if environ.get('MO_ENABLED_TRANSFORMS') == 'ANALYSIS_JSON_PRINT':
```

Comment:
I think you should parse MO_ENABLED_TRANSFORMS variable value first by using `get_enabled_and_disabled_transforms` from `mo\utils\class_registration.py`.
after that you need to check `ANALYSIS_JSON_PRINT` in enabled_transforms.

Reply:
done